packages/good-design/src/core/utils/linking.ts

```diff
@@ -50,3 +50,17 @@ export async function openLink(uri: string, target: "_blank" | "_self" = "_blank
 
   return result;
 }
+
+export const tryJson = (value: string | null) => {
```

Comment:
it had to be already added, somewhere else, please check 

packages/sdk-v2/src/sdk/storage/utils.ts
packages/login-sdk/src/loginLinkUtils.ts (parseLoginLink)

you could replace JSON.parse to tryJSON in parseLoginLink